src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs

```diff
@@ -336,17 +375,31 @@ public int TxPowerLevel
         /// <exception cref="NotSupportedException">Thrown when the Bluetooth LE is not supported.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the Bluetooth LE is not enabled.</exception>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated since API level 6. Please use GetServiceSolicitationUuid() method on BluetoothLeDevice.")]
         public IEnumerable<string> ServiceSolictationUuid
         {
             get
             {
-                if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
-                {
-                    return BluetoothLeImplAdapter.Instance.GetScanResultSvcSolicitationUuids(_scanData, _packetType);
-                }
-                return null;
+                return GetServiceSolicitationUuid(_packetType);
             }
         }
+
+        /// <summary>
+        /// Gets the service solicitation UUID list from the scan result information.
+        /// </summary>
+        /// <value> Gets the list of the service solicitation UUIDs.</value>
+        /// <remarks>The Bluetooth must be enabled.</remarks>
+        /// <param name="packetType"> The enumeration for BLE packet type.</param>
+        /// <returns>The service solicitation uuid value</returns>
```

Comment:
It would be better to keep the consistency.
Since UUID is a abbreviation, please use UUID.